cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/realtime/source/JmsSource.java

```diff
@@ -323,22 +323,22 @@ public JmsProvider getJmsProvider() {
    */
   public static class JmsPluginConfig extends PluginConfig {
     @Name(JMS_DESTINATION_NAME)
-    @Description("Name of the destination to get messages")
+    @Description("Name of the destination from which to get messages.")
     private String destinationName;
 
     @Name(JMS_MESSAGES_TO_RECEIVE)
-    @Description("Max number messages should be retrieved per poll. The default value is 50.")
+    @Description("Max number of messages that should be retrieved per poll. The default value is 50.")
```

Comment:
I guess I was thinking of the case where the default was changed, and then we had to know/remember to update the docs. But we have an issue open to automate this in the future, so we can revisit that at that time. Consider this resolved.